boards/arm/blt_plane/blt_plane.dts

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2025 Launch Initiative
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f4/stm32f446Xe.dtsi>
+#include <st/f4/stm32f446r(c-e)tx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Power Module";
+	compatible = "st,stm32f446xx";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart5;
+		zephyr,shell-uart = &uart5;
+
+		storage = &w25q128;
+		logfs = &lfs1;
+	};
+
+	aliases {
+		inabatt = &ina0;
+		ina3v3 = &ina1;
+		ina5v0 = &ina2;
+		vin = &adc1;
+		rtc = &rv8263;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
```

Comment:
if im understanding what shai was saying, some of these dont work
If thats common across all blts, it might be nice to mark the busted ones here

Reply:
Yea I wanted to wait for them to solder other ones to test so we can confirm if this is a single board issue or a common occurrence